src/EFCore.PG/ValueGeneration/Internal/NpgsqlUUID7ValueGenerator.cs

```diff
@@ -0,0 +1,108 @@
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+namespace Npgsql.EntityFrameworkCore.PostgreSQL.ValueGeneration.Internal;
+
+/// <summary>
+///     Generates sequential <see cref="Guid" /> values according to the UUID version 7 specification.
+///     Will be updated to use Guid.CreateVersion7 when available.
+/// </summary>
+public class NpgsqlUUid7ValueGenerator : ValueGenerator<Guid>
```

Comment:
**Typo:** The class name says `UUid`, which should be `Uuid`. The file name says `UUID`, which should also be `Uuid`.

Reply:
Name of class changed, but filename change only worked locally

Reply:
Ah yeah, that annoying issue. Add a digit or something, `*2.cs`, push, then rename to the desired casing and push again.

Reply:
Edit: If we add the Npgsql prefix, the casing change would also work.

Reply:
Hm, did you add the prefix without adjusting the casing, by any chance?

Reply:
File name should be sorted now